README.md

```diff
@@ -404,6 +410,21 @@ CMS are a practical way to setup a website with many features. CMS often come wi
   * [MaterialPoll](https://github.com/nearbycoder/materialpoll) - an opensource strawpoll.me clone for creating random polls. - ([Source code](https://github.com/nearbycoder/materialpoll), [Demo](http://materialpoll.tk))
   * [OpenSondage](https://github.com/leblanc-simon/OpenSondage) - an online service for planning an appointment or make a decision quickly and easily: Make a poll, Define dates or subjects to choose, Send the poll link to your friends or colleagues, Discuss and make a decision - ([Demo](http://framadate.org/), [Example](https://framadate.org/aqg259dth55iuhwm))
 
+## Project Management
```

Comment:
I think we should differentiate between ticketing (as in sysadmin) and bug tracking (as in developer).

How about splitting the Bug and Task trackers category into bug trackers and "task trackers and ticketing systems", moving the latter to the top level and linking to it from software dev?